### DISCUSSION TITLE
[5.x] Ability to duplicate Matrix Block (Entry)

### DISCUSSION THREAD

**Discussion by BAM-Dennis** [craftcms/cms#14905] [Ideas]
When using the "Cards" or "Inline Editing" view of a matrix block, i am not able to duplicate blocks. i am when using the element Index.

# Expected User Journey
1. Create a new Entry
2. Create a Matrix Block Entry
3. Want to Duplicate this exact block multiple times
4. use the settings icon on the right -> Duplicate

# Current User Journey
1. Create a new Entry
2. Create a Matrix Block Entry
3. Want to Duplicate this exact block multiple times
4. Goto Step 3

**Comment by andrewhawkes** [comment]
It would also be nice to be able to copy/paste cut/paste entries/blocks within Matrix fields
